eventstore/query_json.go

```diff
@@ -0,0 +1,232 @@
+package eventstore
```

Comment:
Most of the new code for querys on JSON Mode.
It has some similarities with Go Type Mode, but variable types kind of forced to be different in subtle ways.